mobile/test/common/jni/test_jni_interface.cc

```diff
@@ -13,7 +13,7 @@ extern "C" JNIEXPORT void JNICALL
 Java_io_envoyproxy_envoymobile_engine_testing_TestJni_nativeStartQuicTestServer(JNIEnv* env,
                                                                                 jclass clazz) {
   jni_log("[QTS]", "starting server");
-  start_server(true);
+  start_server(true, false);
```

Comment:
Is this the only place that start_server is called? If so, it appears that it's being called from C++ code, not from Java. As a result, I think we can easily use an enum here (HTTP3, HTTP2, HTTP1_PLAINTEXT) instead of mixing the bool for quic and the bool for https.

That being said, do we ever call it with disable_https = true, currently?

Reply:
Ah good catch, changed.  We currently use HTTP1 mode in swift, and HTTP2 and HTTP3 modes for kotlin